<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WCamerWidget</class>
 <widget class="QWidget" name="WCamerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grpbx_Cam">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>相机界面</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="hlay_Cam"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlay_camoprt">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_single">
          <property name="text">
           <string>采图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_continue">
          <property name="text">
           <string>连续采图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_saveImg">
          <property name="text">
           <string>保存图像</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_pamset">
          <property name="text">
           <string>参数设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_exp">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>曝光</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_exp">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="hSlider_exp">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 170, 0);
selection-color: rgb(255, 255, 0);</string>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_gain">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>增益</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_gain">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="hSlider_gain">
          <property name="styleSheet">
           <string notr="true">color: rgb(24, 217, 255);</string>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hSlider_exp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_exp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_exp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSlider_exp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSlider_gain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_gain</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_gain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSlider_gain</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
